tools/ffmpeg/ffmpeg_converter.xml

```diff
@@ -10,64 +10,77 @@
         ]]>
     </command>
     <inputs>
-        <param name="input_file" type="data" format="mp4,mov,avi,mkv,mp3,wav,flac,ogg" label="Input media file"/>
+        <param name="input_file" type="data" format="mp4,mov,avi,mkv,webm,mp3,wav,flac,ogg,m4a" label="Input media file"/>
         <param name="output_format" type="select" label="Output format">
             <option value="mp4">MP4 (video)</option>
             <option value="mkv">MKV (video)</option>
             <option value="avi">AVI (video)</option>
             <option value="mov">MOV (video)</option>
+            <option value="webm">WebM (video)</option>
             <option value="mp3">MP3 (audio)</option>
             <option value="wav">WAV (audio)</option>
             <option value="flac">FLAC (audio)</option>
             <option value="ogg">OGG (audio)</option>
+            <option value="m4a">M4A (audio)</option>
         </param>
     </inputs>
     <outputs>
-        <data name="output_file" format="ogg" from_work_dir="output.@output_format@" label="Converted file (${output_format})">
+        <data name="output_file" format="ogg" label="Converted file (${output_format})">
             <change_format>
                 <when input="output_format" value="mp4" format="mp4"/>
                 <when input="output_format" value="mkv" format="mkv"/>
                 <when input="output_format" value="avi" format="avi"/>
                 <when input="output_format" value="mov" format="mov"/>
+                <when input="output_format" value="webm" format="webm"/>
                 <when input="output_format" value="mp3" format="mp3"/>
                 <when input="output_format" value="wav" format="wav"/>
                 <when input="output_format" value="flac" format="flac"/>
+                <when input="output_format" value="m4a" format="m4a"/>
             </change_format>
         </data>
     </outputs>
     <tests>
         <test expect_num_outputs="1">
             <param name="input_file" value="SampleVideo_360x240_cut.mp4" ftype="mp4"/>
             <param name="output_format" value="webm"/>
-            <output name="output_file" file="SampleVideo_360x240_cut.webm" ftype="webm"/>
+            <output name="output_file" file="SampleVideo_360x240_cut.webm" ftype="webm" compare="sim_size" delta="20000"/>
```

Comment:
can you please convert that to asserts, specify the size and the delta ... I think this makes it more reviewable. I don't know how big the file is and if such a large delta is justifyable ....